docs/taskbackground.rst

```diff
@@ -35,23 +65,126 @@ The tree of the above event looks something like
+Using the up/nominal/down measurements, an interpolation can be made (for each bin) so that a model prediction can be provided for any parameter value:
+
+.. image:: images/interpolation.png
+  :width: 60%
+  :alt: Example of interpolation for model predictions, obtained from `CERN-OPEN-2012-016 <https://cds.cern.ch/record/1456844>`_.
+  
+The above example was obtained from `CERN-OPEN-2012-016 <https://cds.cern.ch/record/1456844>`_.
+
+Using our (pseudo)data, we run a maximum likelihood fit over all parameters to find the best-fit parameter values. The model will be made to have higher agreement with the data while nuisance parameters stay as close to the nominal values as possible. 
+
+The results of this fit can be seen in the below diagram, which demonstrates how far each parameter was pulled from its nominal value:
+
+.. image:: images/pulls.png
+  :width: 60%
+  :alt: A pull diagram showing the fit results for each parameter with respect to their nominal values.
+
+Here is what the model looks like, before and after the fit is performed:
+
+.. image:: images/fit.png
+  :width: 80%
+  :alt: The histogram of the reconstructed top mass (:math:`m_{bjj}`) before and after maximum likelihood fit.
+
+For more information to statistics in LHC physics, visit `this document <https://arxiv.org/abs/1503.07622>`_, which aided in writing this quick overview.
+
+6. Machine Learning Component
 ---------------------------------------------------------------
-Most modern high energy physics analyses use some form of machine learning (ML), so a machine learning task has been incorporated into the AGC :math:`t\bar{t}` cross-section  measurement to reflect this development. The method described above to reconstruct the top mass attempts to correctly select all three jets on the hadronic side of the collision. Using ML, we can go beyond this task by attempting to correctly assign each jet with its parent parton. This should allow for a more accurate top mass reconstruction as well as access to new observables, such as the angle between the jet on the leptonic side of the collision and the lepton, or the angle between the two W jets.
+Most modern high energy physics analyses use some form of machine learning (ML), so a machine learning task has been incorporated into the AGC :math:`t\bar{t}` cross-section measurement to reflect this development. 
+The method described above to reconstruct the top mass attempts to correctly select all three jets on the hadronic side of the collision. 
+Using ML, we can go beyond this task by attempting to correctly assign each jet with its parent parton. 
+This should allow for a more accurate top mass reconstruction as well as access to new observables, such as the angle between the jet on the leptonic side of the collision and the lepton, or the angle between the two W jets.
+To enable machine learning in the ttbar analysis notebook, one must set ``USE_INFERENCE = True``. 
```

Comment:
Let's also clarify here that this is for the specific IRIS-HEP implementation (to clearly distinguish task & implementation), and this could then also be a link to the implementation on the website.